Index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Proj Survey Form | Nisar</title>
+</head>
+
+<body>
+  <header>
```

Comment:
Good job on making changes in your project using HTML semantic tags 🥇. However, it will be better if you put `<header>` and `<form>` inside the `<main>` tag.

Reply:
I have made changes as per requirement. [you can check here](https://github.com/MNisarAli/Proj-Survey-form/pull/2).